pkg/pubsub/common/common.go

```diff
@@ -0,0 +1,35 @@
+package common
+
+import (
+	"strconv"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var log = logf.Log.WithName("pubsub-common")
+
+func GetBool(input *bool, dft bool) bool {
+	if input == nil {
+		return dft
+	}
+	return *input
+}
+
+func GetString(input string, dft string) string {
```

Comment:
nit: I'd just use "default", not worth the reduction in readability to save 4 characters.